cinema/urls.py

```diff
@@ -1,10 +1,46 @@
-from django.urls import path
+from django.urls import path, include
+from rest_framework import routers
+
+from cinema.views import (
+    GenreList,
+    GenreDetail,
+    ActorList,
+    ActorDetail,
+    CinemaHallViewSet,
+    MovieViewSet
+)
+
+
+cinema_hall_list = CinemaHallViewSet.as_view(
+    actions={
+        "get": "list",
+        "post": "create",
+    }
+)
+cinema_hall_detail = CinemaHallViewSet.as_view(
+    actions={
+        "get": "retrieve",
+        "put": "update",
+        "patch": "partial_update",
+        "delete": "destroy",
+    }
+)
+router = routers.DefaultRouter()
+router.register("movies", MovieViewSet)
 
-from cinema.views import movie_list, movie_detail
 
 urlpatterns = [
-    path("movies/", movie_list, name="movie-list"),
-    path("movies/<int:pk>/", movie_detail, name="movie-detail"),
+    path("genres/", GenreList.as_view(), name="genre-list"),
+    path("genres/<int:pk>/", GenreDetail.as_view(), name="genre-detail"),
+    path("actors/", ActorList.as_view(), name="actor-list"),
+    path("actors/<int:pk>/", ActorDetail.as_view(), name="actor-detail"),
+    path("cinema_halls/", cinema_hall_list, name="cinema_hall-list"),
+    path(
+        "cinema_halls/<int:pk>/",
+        cinema_hall_detail,
+        name="cinema_halls-detail"
```

Comment:
```suggestion
        name="cinema-halls-detail"
```

Reply:
if you use `-`, use it and don't mix things up

Reply:
I looked from the side sameness name
![image](https://user-images.githubusercontent.com/46852739/230590757-9d37bef2-4743-4845-afe0-8f268a334e6d.png)
ok. 
fixed.